servers/rendering_server.h

```diff
@@ -47,6 +47,7 @@ class RenderingServer : public Object {
 	static RenderingServer *singleton;
 
 	int mm_policy;
+	bool render_loop_enabled;
```

Comment:
This should be initialized.

Reply:
This is implicitly initialized from `main.cpp`, but should it be more explicit then? And if yes, should I do it on the header or on the constructor?

Reply:
I think it should be explicitly initialized here, in the header.

Disabling the render loop is an edge use case, so the default should be that it's enabled.

Reply:
Fixed